include/examples/simple_behavior.h

```diff
@@ -0,0 +1,192 @@
+/**
+ * @file simple_behavior.h
```

Comment:
I might expand or chagne the name of this behavior or its class name to include SpinJoints or SpinAllJoints for safety XD
"SpinAllBehavior" for example

Reply:
SimpleSpinAllBehavior maybe?

Reply:
Changed in f4b04a6f22a7cb91e3c61c238b5b25a3686f1eef